packages/snippet-bot/test/fixtures/config/valid-config.yml

```diff
@@ -1 +1,2 @@
-randomBoolean: true
\ No newline at end of file
+ignoreFiles:
```

Comment:
Huh - I'm curious.  What's a situation where we'd detect a region tag, and then ignore it?

Reply:
I think there should be a way to opt out certain files.
For example, I want to exclude readme which contains an example of mismatched region tag.

Reply:
As I did in the last commit